### PR TITLE
Add certman-operator-service

### PR DESCRIPTION
**Pull request by jharrington22**
Add service for certman

cc @tparikh @jmelis

### CODE REVIEW

File: certman-operator-services/certman-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+services:
+- hash: 0a5e2b891727d5996421a3ca5efe91f746c75d7f
```
Comment (jmelis) [inline certman-operator-services/certman-operator.yaml]: let's start with `hash: none`

That will enable staging deployment, but not prod deployment. Once we verify it works in staging, we can just replace `hash: none` with the actual hash we want to deploy.

File: certman-operator-services/certman-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+services:
+- hash: 0a5e2b891727d5996421a3ca5efe91f746c75d7f 
+  name: certman-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
+  url: https://github.com/openshift/certman-operator
```
Comment (jmelis) [inline certman-operator-services/certman-operator.yaml:5]: let's wait until this exists before merging

**Review by jmelis — CHANGES_REQUESTED** [review]
we are missing the certman-operator context in the /config.yaml file

File: certman-operator-services/certman-operator.yaml (hunk 1 above)
Comment (jharrington22) [reply certman-operator-services/certman-operator.yaml]: Fixed

File: certman-operator-services/certman-operator.yaml (hunk 2 above)
Comment (jharrington22) [reply certman-operator-services/certman-operator.yaml:5]: Everything is merged and ready on the certman side

**Comment by jharrington22** [comment]
@jmelis fixed!

**Review by jmelis — APPROVED** [review]

**Pull request merged**